src/main/java/Duke.java

```diff
@@ -1,10 +1,145 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LINE = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+    private static final Scanner input = new Scanner(System.in);
+
+    private static final List<Task> tasks = new ArrayList<Task>();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+
+        String userInput;
+         do {
+            System.out.println(LINE);
+            userInput = input.nextLine().trim();
+            System.out.println(LINE);
+        } while(executeUserCommand(userInput));
+    }
+
+    /**
+     * Prints welcome message for the bot.
+     */
+    private static void printWelcomeMessage() {
+        System.out.println(LINE);
+        System.out.println("                Hewwo! I'm UwU Bot\n"
+                          +"              Hwere to hwelp you out");
+        System.out.println(LINE);
+        System.out.println("What can I dwo fwor ywou?");
+    }
+
+    /**
+     * Chooses the appropriate command base on user's input.
+     * Returns false if user enters "bye" otherwise returns true.
+     *
+     * @param userInput user's input.
+     * @return boolean value.
+     */
+    private static boolean executeUserCommand(String userInput) {
+        String[] userWords = userInput.split(" ", 2);
+        switch(userWords[0]) {
+        case "bye":
```

Comment:
Consider using constants to store the commands.

Reply:
Aite thanks for the tip!